### ISSUE TITLE
[BUG] v4.0.2

### ISSUE THREAD

**Issue by tsvetiligo** [bizley/yii2-migration#98]
Yii v2.0.35
Yii 2 Migration Generator Tool v4.0.2
MySQL 5.7

I don't know how to describe this behavior:

```
<?php

use yii\db\Migration;

class m200519_132335_test1 extends Migration
{

    public function up()
    {
        $tableOptions = null;
        if ($this->db->driverName === 'mysql') {
            $tableOptions = 'CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ENGINE=InnoDB';
        }

        $this->createTable('{{%test1}}', [
            'col1' => $this->bigPrimaryKey()->unsigned(),
            'col2' => $this->string()->notNull(),
            ], $tableOptions);

        $this->createTable('{{%test2}}', [
            'col1' => $this->bigPrimaryKey()->unsigned(),
            ], $tableOptions);
    }

    public function down()
    {
        $this->dropTable('{{%test1}}');
    }
}
```
```
./yii migrate
Yii Migration Tool (based on Yii v2.0.35)

Total 1 new migration to be applied:
	m200519_132335_test1

Apply the above migration? (yes|no) [no]:yes
*** applying m200519_132335_test1
    > create table {{%test1}} ... done (time: 0.015s)
    > create table {{%test2}} ... done (time: 0.014s)
*** applied m200519_132335_test1 (time: 0.045s)


1 migration was applied.

Migrated up successfully.
```
```
./yii migration/update "*"
Yii 2 Migration Generator Tool v4.0.2
 > 1 table excluded by the config.
 > Are you sure you want to generate migrations for the following tables?
   - test1
   - test2 (yes|no) [no]:yes

 > Comparing current table 'test1' with its migrations ...TABLE IS UP-TO-DATE.

 > Comparing current table 'test2' with its migrations ...DONE!

 > Generating migration for updating table 'test2' ...DONE!
 > Saved as '/srv/migrations/m200618_131402_update_table_test2.php'

 Generated 1 file
 (!) Remember to verify files before applying migration.
```
```
<?php

use yii\db\Migration;

class m200618_131402_update_table_test2 extends Migration
{
    public function up()
    {
        $this->dropColumn('{{%test2}}', 'col2');
    }

    public function down()
    {
        $this->addColumn('{{%test2}}', 'col2', $this->string()->notNull());
    }
}
```

**Comment by bizley** [comment]
I can see everything is fine when table is updated directly. Investigating...

**Comment by bizley** [comment]
Inspector is not resetting applied changes.

**Comment by bizley** [comment]
This is interesting... Applied fix triggers PHPStan error, which is obviously incorrect... FML. I will report it to PHPStan and try to figure out some workaround for now.

**Comment by bizley** [comment]
Fixed in [4.0.3](https://github.com/bizley/yii2-migration/releases/tag/4.0.3)
